cookiecutter.json

```diff
@@ -14,5 +14,7 @@
   "continuous_integration": ["GitHub", "GitLab"],
   "docstring_style": ["NumPy", "Google"],
   "private_package_repository_name": "",
-  "private_package_repository_url": ""
+  "private_package_repository_url": "",
+  "__package_name_kebab_case": "{{ cookiecutter.package_name|slugify }}",
+  "__package_name_snake_case": "{{ cookiecutter.package_name|slugify(separator='_') }}"
```

Comment:
FYI:
1. These are private Cookiecutter parameters [1] – i.e. they are hidden from the user during `cruft create`.
2. We need these because neither `|` nor `"` are legal characters in filepaths on Windows.

[1] https://cookiecutter.readthedocs.io/en/stable/advanced/private_variables.html#private-variables